### DISCUSSION TITLE
Is it possible to suppress all `chown` calls?

### DISCUSSION THREAD

**Discussion by khuongduybui** [tiangolo/uwsgi-nginx-flask-docker#371] [Questions] [answered]
Hi all,

I'm trying to run a container built upon this base image on a platform where `chown` is not permitted.

Is there a way to suppress all these calls, from both `uwsgi` and `nginx`?

**Comment by tiangolo** [comment, marked as answer]
I think there wouldn't be an easy way in this image, that would probably justify building your own. :sweat_smile:
